eng/dogfood.ps1

```diff
@@ -33,6 +33,7 @@ if ($help -or (($command -ne $null) -and ($command.Contains("/help") -or $comman
 try {
   InitializeTools
 
+  $env:PATH = "$RepoRoot\.dotnet;$env:PATH"
```

Comment:
>$RepoRoot\.dotnet [](start = 15, length = 17)

I'd avoid hardcoding the path. You can call `InitializeDotNetCli` function to get the directory if you're on the latest Arcade.

Reply:
Fixed

Reply:
Now reverted. Unnecessary with change to arcade to set PATH